{"message":"ResourceRequest timed out","stack":"TimeoutError: ResourceRequest timed out\n    at ResourceRequest._fireTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\node_modules\\generic-pool\\lib\\ResourceRequest.js:62:17)\n    at Timeout.bound (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\node_modules\\generic-pool\\lib\\ResourceRequest.js:8:15)\n    at ontimeout (timers.js:498:11)\n    at tryOnTimeout (timers.js:323:5)\n    at Timer.listOnTimeout (timers.js:290:5)","name":"TimeoutError","level":"error","timestamp":"2019-04-20T23:06:46.803Z"}
{"message":"ResourceRequest timed out","stack":"TimeoutError: ResourceRequest timed out\n    at ResourceRequest._fireTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\node_modules\\generic-pool\\lib\\ResourceRequest.js:62:17)\n    at Timeout.bound (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\node_modules\\generic-pool\\lib\\ResourceRequest.js:8:15)\n    at ontimeout (timers.js:498:11)\n    at tryOnTimeout (timers.js:323:5)\n    at Timer.listOnTimeout (timers.js:290:5)","name":"TimeoutError","level":"error","timestamp":"2019-04-20T23:07:17.982Z"}
{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","stack":"SequelizeConnectionError: Failed to connect to SQL6003.site4now.net:1433 in 15000ms\n    at Connection.connection.on.err (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\connection-manager.js:113:20)\n    at emitOne (events.js:116:13)\n    at Connection.emit (events.js:211:7)\n    at Connection.connectTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:944:12)\n    at Timeout._onTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:913:16)\n    at ontimeout (timers.js:498:11)\n    at tryOnTimeout (timers.js:323:5)\n    at Timer.listOnTimeout (timers.js:290:5)","name":"SequelizeConnectionError","parent":{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","code":"ETIMEOUT"},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2019-04-20T23:09:05.776Z"}
{"message":"ResourceRequest timed out","stack":"TimeoutError: ResourceRequest timed out\n    at ResourceRequest._fireTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\node_modules\\generic-pool\\lib\\ResourceRequest.js:62:17)\n    at Timeout.bound (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\node_modules\\generic-pool\\lib\\ResourceRequest.js:8:15)\n    at ontimeout (timers.js:498:11)\n    at tryOnTimeout (timers.js:323:5)\n    at Timer.listOnTimeout (timers.js:290:5)","name":"TimeoutError","level":"error","timestamp":"2019-04-20T23:09:18.418Z"}
{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","stack":"SequelizeConnectionError: Failed to connect to SQL6003.site4now.net:1433 in 15000ms\n    at Connection.connection.on.err (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\connection-manager.js:113:20)\n    at emitOne (events.js:116:13)\n    at Connection.emit (events.js:211:7)\n    at Connection.connectTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:944:12)\n    at Timeout._onTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:913:16)\n    at ontimeout (timers.js:498:11)\n    at tryOnTimeout (timers.js:323:5)\n    at Timer.listOnTimeout (timers.js:290:5)","name":"SequelizeConnectionError","parent":{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","code":"ETIMEOUT"},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2019-04-20T23:09:23.490Z"}
{"message":"ResourceRequest timed out","stack":"TimeoutError: ResourceRequest timed out\n    at ResourceRequest._fireTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\node_modules\\generic-pool\\lib\\ResourceRequest.js:62:17)\n    at Timeout.bound (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\node_modules\\generic-pool\\lib\\ResourceRequest.js:8:15)\n    at ontimeout (timers.js:498:11)\n    at tryOnTimeout (timers.js:323:5)\n    at Timer.listOnTimeout (timers.js:290:5)","name":"TimeoutError","level":"error","timestamp":"2019-04-20T23:09:36.008Z"}
{"message":"ResourceRequest timed out","stack":"TimeoutError: ResourceRequest timed out\n    at ResourceRequest._fireTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\node_modules\\generic-pool\\lib\\ResourceRequest.js:62:17)\n    at Timeout.bound (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\node_modules\\generic-pool\\lib\\ResourceRequest.js:8:15)\n    at ontimeout (timers.js:498:11)\n    at tryOnTimeout (timers.js:323:5)\n    at Timer.listOnTimeout (timers.js:290:5)","name":"TimeoutError","level":"error","timestamp":"2019-04-20T23:10:04.728Z"}
{"message":"ResourceRequest timed out","stack":"TimeoutError: ResourceRequest timed out\n    at ResourceRequest._fireTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\node_modules\\generic-pool\\lib\\ResourceRequest.js:62:17)\n    at Timeout.bound (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\node_modules\\generic-pool\\lib\\ResourceRequest.js:8:15)\n    at ontimeout (timers.js:498:11)\n    at tryOnTimeout (timers.js:323:5)\n    at Timer.listOnTimeout (timers.js:290:5)","name":"TimeoutError","level":"error","timestamp":"2019-04-20T23:10:07.326Z"}
{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","stack":"SequelizeConnectionError: Failed to connect to SQL6003.site4now.net:1433 in 15000ms\n    at Connection.connection.on.err (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\connection-manager.js:113:20)\n    at emitOne (events.js:116:13)\n    at Connection.emit (events.js:211:7)\n    at Connection.connectTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:944:12)\n    at Timeout._onTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:913:16)\n    at ontimeout (timers.js:498:11)\n    at tryOnTimeout (timers.js:323:5)\n    at Timer.listOnTimeout (timers.js:290:5)","name":"SequelizeConnectionError","parent":{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","code":"ETIMEOUT"},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2019-04-20T23:10:10.439Z"}
{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","stack":"SequelizeConnectionError: Failed to connect to SQL6003.site4now.net:1433 in 15000ms\n    at Connection.connection.on.err (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\connection-manager.js:113:20)\n    at emitOne (events.js:116:13)\n    at Connection.emit (events.js:211:7)\n    at Connection.connectTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:944:12)\n    at Timeout._onTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:913:16)\n    at ontimeout (timers.js:498:11)\n    at tryOnTimeout (timers.js:323:5)\n    at Timer.listOnTimeout (timers.js:290:5)","name":"SequelizeConnectionError","parent":{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","code":"ETIMEOUT"},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2019-04-20T23:10:12.451Z"}
{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","stack":"SequelizeConnectionError: Failed to connect to SQL6003.site4now.net:1433 in 15000ms\n    at Connection.connection.on.err (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\connection-manager.js:113:20)\n    at emitOne (events.js:116:13)\n    at Connection.emit (events.js:211:7)\n    at Connection.connectTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:944:12)\n    at Timeout._onTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:913:16)\n    at ontimeout (timers.js:498:11)\n    at tryOnTimeout (timers.js:323:5)\n    at Timer.listOnTimeout (timers.js:290:5)","name":"SequelizeConnectionError","parent":{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","code":"ETIMEOUT"},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2019-04-20T23:11:19.297Z"}
{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","stack":"SequelizeConnectionError: Failed to connect to SQL6003.site4now.net:1433 in 15000ms\n    at Connection.connection.on.err (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\connection-manager.js:113:20)\n    at emitOne (events.js:116:13)\n    at Connection.emit (events.js:211:7)\n    at Connection.connectTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:944:12)\n    at Timeout._onTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:913:16)\n    at ontimeout (timers.js:498:11)\n    at tryOnTimeout (timers.js:323:5)\n    at Timer.listOnTimeout (timers.js:290:5)","name":"SequelizeConnectionError","parent":{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","code":"ETIMEOUT"},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2019-04-20T23:11:19.298Z"}
{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","stack":"SequelizeConnectionError: Failed to connect to SQL6003.site4now.net:1433 in 15000ms\n    at Connection.connection.on.err (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\connection-manager.js:113:20)\n    at emitOne (events.js:116:13)\n    at Connection.emit (events.js:211:7)\n    at Connection.connectTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:944:12)\n    at Timeout._onTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:913:16)\n    at ontimeout (timers.js:498:11)\n    at tryOnTimeout (timers.js:323:5)\n    at Timer.listOnTimeout (timers.js:290:5)","name":"SequelizeConnectionError","parent":{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","code":"ETIMEOUT"},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2019-04-20T23:11:34.318Z"}
{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","stack":"SequelizeConnectionError: Failed to connect to SQL6003.site4now.net:1433 in 15000ms\n    at Connection.connection.on.err (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\connection-manager.js:113:20)\n    at emitOne (events.js:116:13)\n    at Connection.emit (events.js:211:7)\n    at Connection.connectTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:944:12)\n    at Timeout._onTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:913:16)\n    at ontimeout (timers.js:498:11)\n    at tryOnTimeout (timers.js:323:5)\n    at Timer.listOnTimeout (timers.js:290:5)","name":"SequelizeConnectionError","parent":{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","code":"ETIMEOUT"},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2019-04-20T23:11:34.318Z"}
{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","stack":"SequelizeConnectionError: Failed to connect to SQL6003.site4now.net:1433 in 15000ms\n    at Connection.connection.on.err (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\connection-manager.js:113:20)\n    at emitOne (events.js:116:13)\n    at Connection.emit (events.js:211:7)\n    at Connection.connectTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:944:12)\n    at Timeout._onTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:913:16)\n    at ontimeout (timers.js:498:11)\n    at tryOnTimeout (timers.js:323:5)\n    at Timer.listOnTimeout (timers.js:290:5)","name":"SequelizeConnectionError","parent":{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","code":"ETIMEOUT"},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2019-04-20T23:11:34.319Z"}
{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","stack":"SequelizeConnectionError: Failed to connect to SQL6003.site4now.net:1433 in 15000ms\n    at Connection.connection.on.err (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\connection-manager.js:113:20)\n    at emitOne (events.js:116:13)\n    at Connection.emit (events.js:211:7)\n    at Connection.connectTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:944:12)\n    at Timeout._onTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:913:16)\n    at ontimeout (timers.js:498:11)\n    at tryOnTimeout (timers.js:323:5)\n    at Timer.listOnTimeout (timers.js:290:5)","name":"SequelizeConnectionError","parent":{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","code":"ETIMEOUT"},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2019-04-20T23:11:34.320Z"}
{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","stack":"SequelizeConnectionError: Failed to connect to SQL6003.site4now.net:1433 in 15000ms\n    at Connection.connection.on.err (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\connection-manager.js:113:20)\n    at emitOne (events.js:116:13)\n    at Connection.emit (events.js:211:7)\n    at Connection.connectTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:944:12)\n    at Timeout._onTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:913:16)\n    at ontimeout (timers.js:498:11)\n    at tryOnTimeout (timers.js:323:5)\n    at Timer.listOnTimeout (timers.js:290:5)","name":"SequelizeConnectionError","parent":{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","code":"ETIMEOUT"},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2019-04-20T23:11:34.322Z"}
{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","stack":"SequelizeConnectionError: Failed to connect to SQL6003.site4now.net:1433 in 15000ms\n    at Connection.connection.on.err (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\connection-manager.js:113:20)\n    at emitOne (events.js:116:13)\n    at Connection.emit (events.js:211:7)\n    at Connection.connectTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:944:12)\n    at Timeout._onTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:913:16)\n    at ontimeout (timers.js:498:11)\n    at tryOnTimeout (timers.js:323:5)\n    at Timer.listOnTimeout (timers.js:290:5)","name":"SequelizeConnectionError","parent":{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","code":"ETIMEOUT"},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2019-04-20T23:11:49.715Z"}
{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","stack":"SequelizeConnectionError: Failed to connect to SQL6003.site4now.net:1433 in 15000ms\n    at Connection.connection.on.err (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\connection-manager.js:113:20)\n    at emitOne (events.js:116:13)\n    at Connection.emit (events.js:211:7)\n    at Connection.connectTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:944:12)\n    at Timeout._onTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:913:16)\n    at ontimeout (timers.js:498:11)\n    at tryOnTimeout (timers.js:323:5)\n    at Timer.listOnTimeout (timers.js:290:5)","name":"SequelizeConnectionError","parent":{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","code":"ETIMEOUT"},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2019-04-20T23:11:49.717Z"}
{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","stack":"SequelizeConnectionError: Failed to connect to SQL6003.site4now.net:1433 in 15000ms\n    at Connection.connection.on.err (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\connection-manager.js:113:20)\n    at emitOne (events.js:116:13)\n    at Connection.emit (events.js:211:7)\n    at Connection.connectTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:944:12)\n    at Timeout._onTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:913:16)\n    at ontimeout (timers.js:498:11)\n    at tryOnTimeout (timers.js:323:5)\n    at Timer.listOnTimeout (timers.js:290:5)","name":"SequelizeConnectionError","parent":{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","code":"ETIMEOUT"},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2019-04-20T23:12:15.083Z"}
{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","stack":"SequelizeConnectionError: Failed to connect to SQL6003.site4now.net:1433 in 15000ms\n    at Connection.connection.on.err (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\connection-manager.js:113:20)\n    at emitOne (events.js:116:13)\n    at Connection.emit (events.js:211:7)\n    at Connection.connectTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:944:12)\n    at Timeout._onTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:913:16)\n    at ontimeout (timers.js:498:11)\n    at tryOnTimeout (timers.js:323:5)\n    at Timer.listOnTimeout (timers.js:290:5)","name":"SequelizeConnectionError","parent":{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","code":"ETIMEOUT"},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2019-04-20T23:12:15.085Z"}
{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","stack":"SequelizeConnectionError: Failed to connect to SQL6003.site4now.net:1433 in 15000ms\n    at Connection.connection.on.err (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\connection-manager.js:113:20)\n    at emitOne (events.js:116:13)\n    at Connection.emit (events.js:211:7)\n    at Connection.connectTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:944:12)\n    at Timeout._onTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:913:16)\n    at ontimeout (timers.js:498:11)\n    at tryOnTimeout (timers.js:323:5)\n    at Timer.listOnTimeout (timers.js:290:5)","name":"SequelizeConnectionError","parent":{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","code":"ETIMEOUT"},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2019-04-20T23:12:30.462Z"}
{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","stack":"SequelizeConnectionError: Failed to connect to SQL6003.site4now.net:1433 in 15000ms\n    at Connection.connection.on.err (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\connection-manager.js:113:20)\n    at emitOne (events.js:116:13)\n    at Connection.emit (events.js:211:7)\n    at Connection.connectTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:944:12)\n    at Timeout._onTimeout (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:913:16)\n    at ontimeout (timers.js:498:11)\n    at tryOnTimeout (timers.js:323:5)\n    at Timer.listOnTimeout (timers.js:290:5)","name":"SequelizeConnectionError","parent":{"message":"Failed to connect to SQL6003.site4now.net:1433 in 15000ms","code":"ETIMEOUT"},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2019-04-20T23:12:30.462Z"}
{"message":"Timeout: Request failed to complete in 15000ms","stack":"SequelizeDatabaseError: Timeout: Request failed to complete in 15000ms\n    at Query.formatError (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\query.js:315:12)\n    at Request.connection.lib.Request [as userCallback] (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\query.js:107:25)\n    at Request._this.callback (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\request.js:61:27)\n    at Connection.message (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:1975:24)\n    at Connection.dispatchEvent (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:1012:38)\n    at MessageIO.<anonymous> (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:891:18)\n    at emitNone (events.js:106:13)\n    at MessageIO.emit (events.js:208:7)\n    at Message.<anonymous> (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\message-io.js:60:15)\n    at emitNone (events.js:111:20)\n    at Message.emit (events.js:208:7)\n    at endReadableNT (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_readable.js:1010:12)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickDomainCallback (internal/process/next_tick.js:218:9)","name":"SequelizeDatabaseError","parent":{"message":"Timeout: Request failed to complete in 15000ms","code":"ETIMEOUT","sql":"WITH \n                  cities_prep as\n                        (SELECT Cities.* ,\n                        0 aS RANK\n                        FROM Cities\n                        WHERE region_id = ISNULL(NULLIF(34091,''),region_id)\n                        ),search_fts as (\n            SELECT t.* FROM (SELECT \n              [KEY],\n              ([RANK])*100/(1+CAST( SUM([RANK]) OVER( PARTITION BY 1) AS FLOAT)) AS [RANK],\n\t\t\t  COUNT(*) OVER() as counter\n              FROM \n              CONTAINSTABLE (Cities,  \n                name_clob,\n                N'\"*\"',\n                LANGUAGE 'english'\n                --\t20  \n              )  \t\t\n\t\t\t) as t\n              WHERE \n\t\t\t  RANK > 70/(case when counter = 0 then 1\n\t\t\t  else  counter end )\n\t\t\t\t\t\n          )\n        \n                  SELECT c.id,c.name,c.longitude,c.latitude FROM cities_prep c\n                  ORDER BY name"},"original":{"$ref":"$[\"parent\"]"},"sql":"WITH \n                  cities_prep as\n                        (SELECT Cities.* ,\n                        0 aS RANK\n                        FROM Cities\n                        WHERE region_id = ISNULL(NULLIF(34091,''),region_id)\n                        ),search_fts as (\n            SELECT t.* FROM (SELECT \n              [KEY],\n              ([RANK])*100/(1+CAST( SUM([RANK]) OVER( PARTITION BY 1) AS FLOAT)) AS [RANK],\n\t\t\t  COUNT(*) OVER() as counter\n              FROM \n              CONTAINSTABLE (Cities,  \n                name_clob,\n                N'\"*\"',\n                LANGUAGE 'english'\n                --\t20  \n              )  \t\t\n\t\t\t) as t\n              WHERE \n\t\t\t  RANK > 70/(case when counter = 0 then 1\n\t\t\t  else  counter end )\n\t\t\t\t\t\n          )\n        \n                  SELECT c.id,c.name,c.longitude,c.latitude FROM cities_prep c\n                  ORDER BY name","level":"error","timestamp":"2019-04-20T23:24:37.186Z"}
{"message":"Timeout: Request failed to complete in 15000ms","stack":"SequelizeDatabaseError: Timeout: Request failed to complete in 15000ms\n    at Query.formatError (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\query.js:315:12)\n    at Request.connection.lib.Request [as userCallback] (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\query.js:107:25)\n    at Request._this.callback (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\request.js:61:27)\n    at Connection.message (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:1975:24)\n    at Connection.dispatchEvent (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:1012:38)\n    at MessageIO.<anonymous> (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:891:18)\n    at emitNone (events.js:106:13)\n    at MessageIO.emit (events.js:208:7)\n    at Message.<anonymous> (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\message-io.js:60:15)\n    at emitNone (events.js:111:20)\n    at Message.emit (events.js:208:7)\n    at endReadableNT (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_readable.js:1010:12)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickDomainCallback (internal/process/next_tick.js:218:9)","name":"SequelizeDatabaseError","parent":{"message":"Timeout: Request failed to complete in 15000ms","code":"ETIMEOUT","sql":"WITH \n                  cities_prep as\n                        (SELECT Cities.* ,\n                        0 aS RANK\n                        FROM Cities\n                        WHERE region_id = ISNULL(NULLIF(34093,''),region_id)\n                        ),search_fts as (\n            SELECT t.* FROM (SELECT \n              [KEY],\n              ([RANK])*100/(1+CAST( SUM([RANK]) OVER( PARTITION BY 1) AS FLOAT)) AS [RANK],\n\t\t\t  COUNT(*) OVER() as counter\n              FROM \n              CONTAINSTABLE (Cities,  \n                name_clob,\n                N'\"*\"',\n                LANGUAGE 'english'\n                --\t20  \n              )  \t\t\n\t\t\t) as t\n              WHERE \n\t\t\t  RANK > 70/(case when counter = 0 then 1\n\t\t\t  else  counter end )\n\t\t\t\t\t\n          )\n        \n                  SELECT c.id,c.name,c.longitude,c.latitude FROM cities_prep c\n                  ORDER BY name"},"original":{"$ref":"$[\"parent\"]"},"sql":"WITH \n                  cities_prep as\n                        (SELECT Cities.* ,\n                        0 aS RANK\n                        FROM Cities\n                        WHERE region_id = ISNULL(NULLIF(34093,''),region_id)\n                        ),search_fts as (\n            SELECT t.* FROM (SELECT \n              [KEY],\n              ([RANK])*100/(1+CAST( SUM([RANK]) OVER( PARTITION BY 1) AS FLOAT)) AS [RANK],\n\t\t\t  COUNT(*) OVER() as counter\n              FROM \n              CONTAINSTABLE (Cities,  \n                name_clob,\n                N'\"*\"',\n                LANGUAGE 'english'\n                --\t20  \n              )  \t\t\n\t\t\t) as t\n              WHERE \n\t\t\t  RANK > 70/(case when counter = 0 then 1\n\t\t\t  else  counter end )\n\t\t\t\t\t\n          )\n        \n                  SELECT c.id,c.name,c.longitude,c.latitude FROM cities_prep c\n                  ORDER BY name","level":"error","timestamp":"2019-04-20T23:26:08.242Z"}
{"message":"The ROLLBACK TRANSACTION request has no corresponding BEGIN TRANSACTION.","stack":"SequelizeDatabaseError: The ROLLBACK TRANSACTION request has no corresponding BEGIN TRANSACTION.\n    at Query.formatError (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\query.js:315:12)\n    at Request.connection.rollbackTransaction.err [as userCallback] (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\query.js:82:25)\n    at Request._this.callback (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\request.js:61:27)\n    at Connection.endOfMessageMarkerReceived (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:1944:20)\n    at Connection.dispatchEvent (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:1012:38)\n    at Parser.<anonymous> (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:812:18)\n    at emitOne (events.js:116:13)\n    at Parser.emit (events.js:211:7)\n    at Parser.<anonymous> (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\token\\token-stream-parser.js:54:15)\n    at emitOne (events.js:116:13)\n    at Parser.emit (events.js:211:7)\n    at addChunk (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_readable.js:291:12)\n    at readableAddChunk (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_readable.js:278:11)\n    at Parser.Readable.push (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_readable.js:245:10)\n    at Parser.Transform.push (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_transform.js:148:32)\n    at Parser.afterTransform (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_transform.js:91:10)\n    at Parser._transform (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\token\\stream-parser.js:69:9)\n    at Parser.Transform._read (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_transform.js:184:10)\n    at Parser.Transform._write (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_transform.js:172:83)\n    at doWrite (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_writable.js:428:64)\n    at writeOrBuffer (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_writable.js:417:5)\n    at Parser.Writable.write (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_writable.js:334:11)","name":"SequelizeDatabaseError","parent":{"message":"The ROLLBACK TRANSACTION request has no corresponding BEGIN TRANSACTION.","code":"EREQUEST","number":3903,"state":2,"class":16,"serverName":"LAPTOP-OOE6F2CH\\SQLEXPRESS","procName":"","lineNumber":1},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2019-04-20T23:42:07.483Z"}
{"message":"The ROLLBACK TRANSACTION request has no corresponding BEGIN TRANSACTION.","stack":"SequelizeDatabaseError: The ROLLBACK TRANSACTION request has no corresponding BEGIN TRANSACTION.\n    at Query.formatError (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\query.js:315:12)\n    at Request.connection.rollbackTransaction.err [as userCallback] (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\query.js:82:25)\n    at Request._this.callback (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\request.js:61:27)\n    at Connection.endOfMessageMarkerReceived (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:1944:20)\n    at Connection.dispatchEvent (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:1012:38)\n    at Parser.<anonymous> (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:812:18)\n    at emitOne (events.js:116:13)\n    at Parser.emit (events.js:211:7)\n    at Parser.<anonymous> (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\token\\token-stream-parser.js:54:15)\n    at emitOne (events.js:116:13)\n    at Parser.emit (events.js:211:7)\n    at addChunk (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_readable.js:291:12)\n    at readableAddChunk (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_readable.js:278:11)\n    at Parser.Readable.push (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_readable.js:245:10)\n    at Parser.Transform.push (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_transform.js:148:32)\n    at Parser.afterTransform (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_transform.js:91:10)\n    at Parser._transform (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\token\\stream-parser.js:69:9)\n    at Parser.Transform._read (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_transform.js:184:10)\n    at Parser.Transform._write (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_transform.js:172:83)\n    at doWrite (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_writable.js:428:64)\n    at writeOrBuffer (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_writable.js:417:5)\n    at Parser.Writable.write (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_writable.js:334:11)","name":"SequelizeDatabaseError","parent":{"message":"The ROLLBACK TRANSACTION request has no corresponding BEGIN TRANSACTION.","code":"EREQUEST","number":3903,"state":2,"class":16,"serverName":"LAPTOP-OOE6F2CH\\SQLEXPRESS","procName":"","lineNumber":1},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2019-04-20T23:47:00.585Z"}
{"message":"The ROLLBACK TRANSACTION request has no corresponding BEGIN TRANSACTION.","stack":"SequelizeDatabaseError: The ROLLBACK TRANSACTION request has no corresponding BEGIN TRANSACTION.\n    at Query.formatError (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\query.js:315:12)\n    at Request.connection.rollbackTransaction.err [as userCallback] (C:\\Programowanie\\Private\\stuffshare\\node_modules\\sequelize\\lib\\dialects\\mssql\\query.js:82:25)\n    at Request._this.callback (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\request.js:61:27)\n    at Connection.endOfMessageMarkerReceived (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:1944:20)\n    at Connection.dispatchEvent (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:1012:38)\n    at Parser.<anonymous> (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\connection.js:812:18)\n    at emitOne (events.js:116:13)\n    at Parser.emit (events.js:211:7)\n    at Parser.<anonymous> (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\token\\token-stream-parser.js:54:15)\n    at emitOne (events.js:116:13)\n    at Parser.emit (events.js:211:7)\n    at addChunk (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_readable.js:291:12)\n    at readableAddChunk (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_readable.js:278:11)\n    at Parser.Readable.push (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_readable.js:245:10)\n    at Parser.Transform.push (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_transform.js:148:32)\n    at Parser.afterTransform (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_transform.js:91:10)\n    at Parser._transform (C:\\Programowanie\\Private\\stuffshare\\node_modules\\tedious\\lib\\token\\stream-parser.js:69:9)\n    at Parser.Transform._read (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_transform.js:184:10)\n    at Parser.Transform._write (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_transform.js:172:83)\n    at doWrite (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_writable.js:428:64)\n    at writeOrBuffer (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_writable.js:417:5)\n    at Parser.Writable.write (C:\\Programowanie\\Private\\stuffshare\\node_modules\\readable-stream\\lib\\_stream_writable.js:334:11)","name":"SequelizeDatabaseError","parent":{"message":"The ROLLBACK TRANSACTION request has no corresponding BEGIN TRANSACTION.","code":"EREQUEST","number":3903,"state":2,"class":16,"serverName":"LAPTOP-OOE6F2CH\\SQLEXPRESS","procName":"","lineNumber":1},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2019-04-20T23:49:44.075Z"}
{"message":"ENOENT: no such file or directory, open 'C:\\Programowanie\\Private\\stuffshare\\src\\Back-end\\Static\\MailsXSLT\\authorization..xslt'","stack":"Error: ENOENT: no such file or directory, open 'C:\\Programowanie\\Private\\stuffshare\\src\\Back-end\\Static\\MailsXSLT\\authorization..xslt'\n    at Object.fs.openSync (fs.js:646:18)\n    at Object.fs.readFileSync (fs.js:551:33)\n    at MailSender.readXSLT (C:/Programowanie/Private/stuffshare/src/Back-end/Architecture/mailSender.js:32:15)\n    at MailSender.mailSend (C:/Programowanie/Private/stuffshare/src/Back-end/Architecture/mailSender.js:19:21)\n    at C:/Programowanie/Private/stuffshare/src/Back-end/Commands/User/createUserCommand.js:69:23\n    at Generator.next (<anonymous>)\n    at step (C:\\Programowanie\\Private\\stuffshare\\src\\Back-end\\Commands\\User\\createUserCommand.js:69:191)\n    at C:\\Programowanie\\Private\\stuffshare\\src\\Back-end\\Commands\\User\\createUserCommand.js:69:361\n    at <anonymous>","errno":-4058,"code":"ENOENT","syscall":"open","path":"C:\\Programowanie\\Private\\stuffshare\\src\\Back-end\\Static\\MailsXSLT\\authorization..xslt","level":"error","timestamp":"2019-04-20T23:51:32.771Z"}
